v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -10,6 +10,7 @@ import {AttributeDescription, DataConfigurationModel, IAttributeDescriptionSnaps
   from "../../data-display/models/data-configuration-model"
 import {GraphAttrRole, graphPlaceToAttrRole, PrimaryAttrRoles} from "../../data-display/data-display-types"
 import {updateCellKey} from "../adornments/adornment-utils"
+import { cachedFnWithArgsFactory } from "../../../utilities/mst-utils"
```

Comment:
I would move this ⬆️ next to `js-utils` so the `../../../utilities` imports are together.

Reply:
Done, thanks.